faker/providers/user_agent/__init__.py

```diff
@@ -5,6 +5,7 @@
 from datetime import datetime
 
 from .. import BaseProvider
+import string
```

Comment:
can you move this to the top of the file? (Above `from datetime import datetime)`

Reply:
it's just a stylistical convention. I like to have imports from the standard lib at the top.